pkg/components/util/install.go

```diff
@@ -35,8 +35,6 @@ func InstallComponent(name string, c components.Component, kubeconfig string) er
 
 // InstallAsRelease installs a component as a Helm release using a Helm client.
 func InstallAsRelease(name string, c components.Component, kubeconfig string) error {
-	actionConfig := &action.Configuration{}
```

Comment:
(commenting here as no other easy way to comment on this commit) **IMHO** `Refactor *`is quite a bad commit title. TONs of things can be "refactors". Why not explain what is being done? Like "Add HelmActionConfig() to improve readability", "cleanup: move common code to Add HelmActionConfig()" (or whatever you think explains better what is being done)

Reply:
Fixed.